src/cms/adobe-stock.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Adobe Stock Integration
+---
+
+Merchants can integrate [Adobe Stock][adobe-stock] with Magento giving them access
+to countless media assets for use in their stores.
+
+![Adobe Stock Search Results]({% link images/images/adobe-stock/search-grid.png %}){: .zoom}
+
+The Adobe Stock service provides businesses with access to millions of high-quality,
+curated, royalty-free photos, vectors, illustrations, videos, templates, and 3D
+assets for all their creative projects. Magento users are able to quickly find,
+preview, and license Adobe Stock assets. Users can also save them to the
+[Media Storage][media-storage], all without leaving the Magento workspace.
+
+## Prerequisites
+
+This integration requires an [adobe.io Console][io-console] account and Magento
+2.3.4 or newer. In order to license [Adobe Stock][adobe-stock] images, a paid plan
+will also be required.
+
+## Integrate Magento and Adobe Stock
+
+It is a two step process in order to configure the Adobe Stock Integration with
+Magento: [create an adobe.io integration][create-integration] in order to create an API Key, and
+then [configure the Adobe Stock Integration inside Magento's System Configuration][configure].
+
+### Create an Adobe.io Integration
+
+1. Navigate to the [adobe.io Console][io-console] and under **Quick Start** click
+   **Create integration**
+2. Ensure **Access an API** is checked and click **Continue**
+3. Under the **Creative Cloud** column, ensure **Creative SDK** is checked and
+   click **Continue**. At this time, _do not_ use the **Adobe Stock** service on
+   this screen.
+4. On the final **Integration Details** screen, fill out the form accordingly:
+    - **Name**: a descriptive name for your integration
+    - **Description**: a longer form description for your integration
+    - **Platform**: Ensure **Web** is checked
+    - **Default redirect URI**: The default redirect URI is in the form `${HOST}/${ADMIN_URI}/adobe_ims/oauth/callback/`
+      (i.e.: `https://store.myshop.com/admin_hgkq1l/adobe_ims/oauth/callback/`), where:
+        - `${HOST}` is your Magento instance's fully qualified domain name (i.e.
+          `https://store.myshop.com`)
+        - `${ADMIN_URI}` is your instance's admin URI which can be retrieved by running
+          `magento info:adminuri` (i.e.: `admin_hgkq1l`)
+    - **Redirect URI pattern**: The redirect URI pattern is the same as your redirect
+      URI with two changes:
+        - Any periods (`.`) need to be escaped with two backslashes (`\\`)
+        - You will need to add `.*` to the end of the pattern
+        - Using the example from the Default redirect URI field above, the
+          Redirect URI pattern would be `https://store\\.myshop\\.com/admin_hgkq1l/adobe_ims/oauth/callback/.*`
```

Comment:
I think this is complicated step from merchant point of view and we can simplify by merchant to copy and paste the `Default redirect URI` and `Redirect URI pattern` from Magento system configuration

Reply:
You can use the formatting for a fenced code block that can be easily copied if needed.

Reply:
I will work on this formatting after merge